packages/substrate/src/fungible.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import type {
+  Domain,
+  FeeHandlerType,
+  SubstrateResource,
+} from "@buildwithsygma/sygma-sdk-core";
+import type { ApiPromise, SubmittableResult } from "@polkadot/api";
+
+export type SubstrateFee = {
+  fee: bigint;
+  type: FeeHandlerType;
+};
+
+/**
+ * Return amount of liqudiity tokens on resource handler
+ * @param provider
+ * @param resource
+ */
+export function getLiquidity(
+  provider: ApiPromise,
+  resource: SubstrateResource,
+): Promise<bigint> {
+  throw new Error("Method not implemented");
+}
+
+type SubstrateAssetTransferRequest = {
+  source: string | number | Domain;
+  sourceNetworkProvider: ApiPromise;
+  destination: string | number | Domain;
+  resource: string | SubstrateResource;
+  amount: bigint;
+  destinationAddress: string;
```

Comment:
To use the init config from "core", we still need to provide the environment similar to EVM. Did I miss something about why the environment was removed? I still need to know it for the "createDestIdMultilocationData" method

Reply:
this has been discussed a few times in the office and we have decided to restore environment parameter in config class, however it should now be set as an environment variable and used from there